compiler/wasm/src/compile.rs

```diff
@@ -136,6 +161,7 @@ pub fn compile(
     entry_point: String,
     contracts: Option<bool>,
```

Comment:
Generally not a fan of constructs like Option<bool> -- I believe it is here so that we can supply a variable number of arguments in javascript land. Though because I put file_source_map as the last argument, it negates the usage 😅

Reply:
Let's leave this as is for now and we can modify this in a followup.